### ISSUE TITLE
False position invalid block usage of multi_users_select

### ISSUE THREAD

**Issue by MechJosh0** [php-slack/slack#74]
I'm attempting to use input type `multi_users_select` but I am being thrown an error that I'm not allowed to use it. I'm assuming the package isn't aware it's a valid input type. It does exist and can be seen in Slacks [Block Kit Buider](https://app.slack.com/block-kit-builder/T0K9XJPSN#%7B%22blocks%22:%5B%7B%22type%22:%22input%22,%22element%22:%7B%22type%22:%22multi_users_select%22,%22placeholder%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Select%20user%22,%22emoji%22:true%7D,%22action_id%22:%22multi_users_select-action%22%7D,%22label%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Select%20users%22,%22emoji%22:true%7D%7D%5D%7D) and [block elements documentation](https://api.slack.com/reference/block-kit/block-elements).

```bash
Invalid Block type \"multi_users_select\". Must be one of: button, checkboxes, datepicker, image, multi_static_select, overflow, plain_text_input, radio_buttons, static_select, plain_text, mrkdwn.
```

**Comment by cmbuckley** [comment]
Turns out this is a larger piece of work than I thought — I’ve created [cmbuckley/slack@new-block-types](https://github.com/cmbuckley/slack/tree/new-block-types) so far, which is still largely untested so I won’t create a PR until I’ve written a few more tests.

**Comment by cmbuckley** [comment]
Hi,

Please try to provide a [self-contained example](http://sscce.org/) of your error, because the block you have posted above works fine for me. You can also see this at the [block kit builder](https://app.slack.com/block-kit-builder/T04LVPZG1#%7B%22type%22:%22home%22,%22blocks%22:%5B%7B%22type%22:%22section%22,%22block_id%22:%22section678%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22Pick%20users%20from%20the%20list%22%7D,%22accessory%22:%7B%22action_id%22:%22text1234%22,%22type%22:%22multi_users_select%22,%22initial_users%22:%5B%5D,%22placeholder%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Select%20users%22%7D%7D%7D%5D%7D).

You can debug the message as follows:

```php
try {
    $client->send();
} catch (GuzzleHttp\Exception\ClientException $ex) {
    var_dump(json_encode($client->preparePayload($client->getMessage())));
}
```

Also, please can you explain more your point about `initial_users`, because again this example appears to work fine without adding the field:

```php
$client =  new Maknz\Slack\Client($token);
$client->withBlock([
    "type" => "section",
    "block_id" => "section678",
    "text" => [
        "type" => "mrkdwn",
        "text" => "Pick users from the list",
    ],
    "accessory" => [
        "action_id" => "text1234",
        "type" => "multi_users_select",
        "placeholder" => [
            "type" => "plain_text",
            "text" => "Select users",
        ],
    ]
])->send('Fallback message');
```

**Comment by alek13** [comment]
@MechJosh0 released in [2.2.1](https://github.com/php-slack/slack/releases/tag/2.2.1)
